src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -45,6 +45,15 @@ namespace Microsoft.VisualStudio.Extensibility.Testing
 {
     internal partial class EditorInProcess
     {
+        public async Task<string> GetDirtyIndicatorAsync(CancellationToken cancellationToken)
+        {
+            var version = await TestServices.Shell.GetVersionAsync(cancellationToken);
+            if (version < Version.Parse("17.2.32224.407"))
```

Comment:
it looks like the PR added an option - to be resilient in case they change the default values for it, should we just have a helper that asserts the name is either dot or star?

Reply:
Interesting, this is a user-visible option too:
![image](https://user-images.githubusercontent.com/1408396/155814198-0312b9c4-c2c3-47a1-8c0b-abfad40389ec.png)